tests/PactNet.Native.Tests/NativePactBuilderTests.cs

```diff
@@ -39,23 +56,147 @@ public void UponReceiving_WhenCalled_CreatesNewInteraction()
         }
 
         [Fact]
-        public void Build_WhenCalled_StartsMockServer()
+        public void Verify_WhenCalled_StartsMockServer()
         {
-            this.builder.Build();
+            this.builder.Verify(Success);
 
             this.mockServer.Verify(s => s.CreateMockServerForPact(this.handle, "127.0.0.1:0", false));
         }
 
         [Fact]
-        public void Build_ErrorStartingMockServer_ThrowsInvalidOperationException()
+        public void Verify_ErrorStartingMockServer_ThrowsInvalidOperationException()
         {
-            this.mockServer.Setup(
-                    s => s.CreateMockServerForPact(It.IsAny<PactHandle>(), It.IsAny<string>(), It.IsAny<bool>()))
+            this.mockServer
+                .Setup(s => s.CreateMockServerForPact(It.IsAny<PactHandle>(), It.IsAny<string>(), It.IsAny<bool>()))
                 .Throws<InvalidOperationException>();
 
-            Action action = () => this.builder.Build();
+            Action action = () => this.builder.Verify(Success);
 
             action.Should().Throw<InvalidOperationException>("because the mock server failed to start");
         }
+
+        [Fact(Skip = "Enable this when the released FFI library supports getting mock server logs")]
+        public void Verify_NoLogs_DoesNotWriteLogs()
+        {
+            this.builder.Verify(Success);
+
+            this.mockOutput.Verify(o => o.WriteLine(It.IsAny<string>()), Times.Never);
+        }
+
+        [Fact(Skip = "Enable this when the released FFI library supports getting mock server logs")]
+        public void Verify_Logs_WritesLogsToOutput()
+        {
+            this.mockServer.Setup(s => s.MockServerLogs(this.serverUri.Port)).Returns("some logs");
```

Comment:
I would use a variable expectedLogs to use line 89 and 93.

Reply:
Introduced local constant for this